kafka/consumer/group.py

```diff
@@ -861,6 +861,48 @@ def metrics(self, raw=False):
             metrics[k.group][k.name] = v.value()
         return metrics
 
+    def offsets_for_times(self, timestamps):
+        """
+        Look up the offsets for the given partitions by timestamp. The returned
```

Comment:
>Look up the offsets for the given partitions...

This confuses me because "given" implies a list of partitions is passed in, but that's not present in the method args. If the partitions are attached to the class, then maybe use wording other than "given" to describe the partitions?